content/en/docs/started/workstation/minikube-linux.md

```diff
@@ -1,15 +1,12 @@
 +++
-title = "Deploy using MiniKube on Linux"
-description = "Install Kubeflow on Minikube on Linux"
+title = "Deploying with MiniKube on a single node"
+description = "How to deploy Kubeflow with MiniKube on a single node on Linux"
```

Comment:
```suggestion
title = "Deploying with miniKube on a single node"
description = "How to deploy Kubeflow with miniKube on a single node on Linux"
```

For consistency.

I guess K8s should also do the same: https://kubernetes.io/docs/tasks/tools/install-minikube/ https://github.com/kubernetes/minikube - at the moment it's both :)